### PR TITLE
EDLY-2185: set height of video element to 607 because its width is 1080

### PR DESCRIPTION
**Pull request by ziafazal**
set height of video element to 607 because its width is 1080.According to aspect ratio of 16:9 height should be 607px.

### CODE REVIEW

**Review by rehan99000 — COMMENTED** [review]
Has this been tested on different screen sizes?

**Comment by ziafazal** [comment]
@rehan99000 this xblock is not responsive and these changes are to fix issue when videos are rendered on 1080 or greater widths.

File: video_xblock/static/html/youtube.html [hunk 1]
```diff
@@ -2,10 +2,9 @@
   id="{{ video_player_id }}"
   class="video-js vjs-default-skin"
   controls
-  height="560"
+  height="607"
```
Comment (zubair-arbi) [inline video_xblock/static/html/youtube.html:5]: I believe Youtube worked fine before this change.

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

File: video_xblock/static/html/youtube.html (hunk 1 above)
Comment (ziafazal) [reply video_xblock/static/html/youtube.html:5]: I faced same issue on Youtube too while testing it.

**Review by rehan99000 — APPROVED** [review]

**Pull request merged**